tests/ZendTest/ServiceManager/ServiceManagerTest.php

```diff
@@ -1189,4 +1189,53 @@ public function testIsSharedThrowsExceptionWhenPassedNameWhichDoesNotExistAnywhe
         $this->setExpectedException('Zend\ServiceManager\Exception\ServiceNotFoundException');
         $this->serviceManager->isShared('foobarbazbat');
     }
+
+    /**
+     * @covers Zend\ServiceManager\ServiceManager::loopPeeringServiceManagers
+     * @covers Zend\ServiceManager\ServiceManager::setServiceManagerCaller
```

Comment:
These two statements are causing PHPUnit errors to occur; it looks like PHPUnit does not like `@covers` for non-public methods: https://travis-ci.org/zendframework/zf2/jobs/53314224#L486

I'll remove the statements during merge.